.github/workflows/update-third-party-notices.yml

```diff
@@ -0,0 +1,42 @@
+name: Automated Linting
+on:
+  workflow_call:
+    secrets:
+      BOT_REPO_SCOPED_TOKEN:
+        required: true
+
+jobs:
+  license-check:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          registry-url: "https://registry.npmjs.org"
+      - run: npm install -g json@v11.0.0
+      - run: npm install -g generate-license-file@2.0.0
```

Comment:
nit: can we install both packages in the same command? npm install should accept multiple package names